components/Gallery.js

```diff
@@ -5,34 +5,34 @@ const Gallery = () => {
         <>
             <div className={styles.row}> 
                 <div className={styles.column}>
-                    <img src="/fam/04.jpg" />
-                    <img src="/fam/01.jpg" />
-                    <img src="/fam/17.png" />
-                    <img src="/fam/06.png" />
-                    <img src="/fam/07.jpg" />
+                    <img src="/fam/04.jpg" alt="" />
```

Comment:
This is great for resolving the `warning`, but let's find some time to put some _real_ content here.

If you're unfamiliar with how to format good `alt` attributes, look no further: https://moz.com/learn/seo/alt-text

Reply:
check